crates/wasmparser/src/validator.rs

```diff
@@ -703,6 +717,17 @@ impl Validator {
         Ok(())
     }
 
+    fn event_type(&self, ty: &EventType) -> Result<()> {
+        if ty.attribute != 0 {
```

Comment:
This is validated currently at both the binary layer and this layer, but should it only be validated at the binary layer?

Reply:
attribute is specified as 0 for this proposal -- I'll remove the `attributes` field for now